packs/orion/tests/fixtures/orion_npm_results.yaml

```diff
@@ -0,0 +1,6 @@
+---
+results:
```

Comment:
On the back edit it's a odd combination of JSON and XML, but the OrionSDK does a very good job of mostly hiding it. 

So the Python module returns dicts, so I used yaml as fixtures and Mocks to turn the data in a sain manner to allow reuse of the data that is repeated often (like queries in `get_node()`) :-D.